project_generation/project_generation.go

```diff
@@ -266,7 +267,8 @@ func (a application) generateBatchOfFileTemplates(filesToGen []fileGen) error {
 
 // generateFileFromTemplate will generate a single file from templates
 func (a application) generateFileFromTemplate(fileToGen fileGen) (err error) {
-	f, err := os.Create(a.pathToRepo + fileToGen.filePrefix + fileToGen.outputPath + fileToGen.extension)
+	outputFilePath := a.pathToRepo + fileToGen.filePrefix + fileToGen.outputPath + fileToGen.extension
+	f, err := os.Create(outputFilePath)
```

Comment:
Do we need/want to check if the file already exists? Possibly overwrite it if it does?

Reply:
There is  previous check to see if the directory we are writing into is empty.  I think there's a lot of opportunity for improving overwriting of existing files in previously generated repos but it's quite a scope creep from this task. Suggest creating an issue on this?